app/ante_handler.go

```diff
@@ -0,0 +1,64 @@
+package gaia
+
+import (
```

Comment:
Why do we have/need a custom ante-handler file/setup?

Reply:
@AdityaSripal? I think the answer is yes if you want the IBC ante handler which discards redundant relay txs. From my understanding the ante handler isn't extensible without redefining it

Reply:
Yes because we're adding an ante decorator (for IBC) that's not part of the SDK default list

Reply:
But this:

> From my understanding the ante handler isn't extensible without redefining it

Do we really have to redefine everything? Can't we just add/replace what we want?